image-files/scripts/run.sh

```diff
@@ -10,6 +11,20 @@ if [ -n "$VNC_SERVER_PASSWORD" ]; then
   /root/scripts/run_x11_vnc.sh &
 fi
 
+# Start either TWS or IB Gateway
```

Comment:
is it more clear to pass a parameter value tws or gateway instead of a command -g?